spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe 'Post', type: :request do
+  describe 'Get post Index' do
+    before :each do
+      get '/users/:user_id/posts/'
+    end
+
+    it 'Returns http status of success for post index' do
+      expect(response).to have_http_status(:ok)
+    end
+
+    it 'Render the right index template for post' do
+      expect(response).to render_template(:index)
+    end
+
+    describe 'Get all related routes to post show template' do
+      before :each do
+        get '/users/:user_id/posts/id'
+      end
+
```

Comment:
- [ ] As I said after commenting that line of code, some tests were not passing and that is the error I am getting 👇 
![image](https://user-images.githubusercontent.com/90222110/214279579-1b02b425-ac90-4d91-93b6-2b4464735373.png)

![image](https://user-images.githubusercontent.com/90222110/214279656-b90b0c08-be64-4d98-ba07-26e68b2e9774.png)

Please do your best to correct the error so that all the tests will be passing 😃 👍